### PR TITLE
Submitting RNAtools

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,80 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not delete the example recipe

File: recipes/RNAtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
```
Comment (synapticarbors) [inline recipes/RNAtools/meta.yaml]: You're going to need `cython` here, I think

File: recipes/RNAtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (synapticarbors) [inline recipes/RNAtools/meta.yaml]: Can you add entry points for the following:

https://github.com/grice/RNAtools/blob/master/setup.py#L23-L28

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

along with tests for the commands:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/example/meta.yaml [hunk 4]
```diff
@@ -1,80 +0,0 @@
...
-
-# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (bgruening) [inline recipes/example/meta.yaml]: please leve the example file in

File: recipes/RNAtools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+about:
+  home: http://github.com/grice/RNAtools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'RNAtools is a Python package for the manipulation of RNA files.'
+
+extra:
+  recipe-maintainers:
```
Comment (synapticarbors) [inline recipes/RNAtools/meta.yaml]: Can you check with the listed co-maintainers to make sure they're ok with being added here, or just let us know if you already had that conversation.

File: recipes/RNAtools/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/RNAtools/meta.yaml]: it looks like there might be additional run time requirements:

https://github.com/grice/RNAtools/blob/2bb1cb42920191d841fe06940081450f69af5dbe/requirements.txt

File: recipes/RNAtools/meta.yaml (hunk 5 above)
Comment (ericmjl) [reply recipes/RNAtools/meta.yaml]: @synapticarbors we've already had that conversation, so no problems here. Thanks for the reminder nonetheless!

File: recipes/example/meta.yaml (hunk 1 above)
Comment (ericmjl) [reply recipes/example/meta.yaml]: Oh, okay. Got it, going to restore.

File: recipes/RNAtools/meta.yaml (hunk 3 above)
Comment (ericmjl) [reply recipes/RNAtools/meta.yaml]: Done!

File: recipes/RNAtools/meta.yaml (hunk 2 above)
Comment (ericmjl) [reply recipes/RNAtools/meta.yaml]: Added!

**Comment by ocefpaf** [comment]
I guess you are covered here @ericmjl :smile:
Thanks @synapticarbors and @bgruening for the review.

**Comment by ericmjl** [comment]
yes, thank you @synapticarbors and @bgruening for your guidance!

File: recipes/RNAtools/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    - dot2ct -h
+    - colorRNA -h
+    - arcsRNA -h
+    - cleanCT -h
+
+about:
+  home: http://github.com/grice/RNAtools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (CurtLH) [inline recipes/RNAtools/meta.yaml]: It looks like the license isn't included in the .tar.gz, so you will need to [package the license manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).

Comment (ericmjl) [reply recipes/RNAtools/meta.yaml]: Thanks for the catch, @CurtLH. I have basically done a version bump which has the license file included.

**Comment by ericmjl** [comment]
Finally, looks like I got the recipe done right! cc: @grice @varunshiva

Just waiting on Travis CI to finish the build.

**Comment by ericmjl** [comment]
@ocefpaf I think we're good here!

File: recipes/python_http_client/meta.yaml [hunk 8]
```diff
@@ -1,44 +0,0 @@
-{% set name = "python_http_client" %}
```
Comment (ocefpaf) [inline recipes/python_http_client/meta.yaml]: There is no need to remove lingering recipes. But in this case it does not hurt b/c those are already feedstocked.

**Pull request merged**